### PR TITLE
News permissions

### PR DESCRIPTION
**Pull request by yaroslavbilokin**

### CODE REVIEW

File: __tests__/news/news.mutation.test.js [hunk 1]
```diff
@@ -60,9 +62,14 @@ const existingNews = {
 };
 
 let newsId = '';
+let token;
 const newsDoesNotExistId = '1f2ad410eb01783384e6111b';
 
 describe('test news mutations', () => {
```
Comment (Alan18081) [inline __tests__/news/news.mutation.test.js:68]: Please, create 3 `describe` statements for `Create` tests, `Update` tests, `Delete` tests to increase the readability of the tests

Comment (Alan18081) [reply __tests__/news/news.mutation.test.js:68]: Please, also add here tests for permissions, that simple users without admin permissions or even without token cannot add, update or delete materials and check that proper error messages are returned.

File: __tests__/news/news.query.test.js [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
 const { NEWS_NOT_FOUND } = require('../../error-messages/news.messages');
 
+require('dotenv').config();
+
 let newsId = '';
+let token;
 const newsDoesNotExistId = '5f311ec5f2983e390432a8c3';
 
 describe('querries', () => {
   beforeAll(async () => {
```
Comment (Alan18081) [inline __tests__/news/news.query.test.js:14]: Please, add here two `describe` statements, one for all tests related to the bulk get, another - for the get by id. Also add tests that both logged-in users, users without token and admins can properly perform bulk get and get by id operations

File: utils/loginAdmin.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+require('dotenv').config();
```
Comment (Alan18081) [inline utils/loginAdmin.js:1]: Please, rename the file to `login-admin`

Comment (yaroslavbilokin) [reply utils/loginAdmin.js:1]: Renamed!

File: __tests__/news/news.mutation.test.js (hunk 1 above)
Comment (yaroslavbilokin) [reply __tests__/news/news.mutation.test.js:68]: You're right! Changed!

File: __tests__/news/news.mutation.test.js [hunk 4]
```diff
@@ -60,10 +66,15 @@ const existingNews = {
 };
 
 let newsId = '';
+let token;
 const newsDoesNotExistId = '1f2ad410eb01783384e6111b';
+const invalidToken =  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI1ZjQ3NTMxYjVjNWYzMTJkMDhlYjdkZDUiLCJlbWFpbCI6ImFkbWluQGdtYWlsLmNvbSIsImlhdCI6MTYwMDA2NTE1AaaaAAaaAaaaAaAaAAA4AaAaaA.A9-aAAaaaaaa-D5ghd96IjsKq_r18tD97WhzI7QXfbA';
 
-describe('test news mutations', () => {
-  test('#1 should add news to database', async () => {
+describe('Create news test', () => {
```
Comment (Alan18081) [inline __tests__/news/news.mutation.test.js:73]: Please, put all of the `describe` statements for news into one singe `describe` with name `News mutations tests`

Comment (yaroslavbilokin) [reply __tests__/news/news.mutation.test.js:73]: Changed!

**Pull request merged**